### ISSUE TITLE
Logs no longer include call stack

### ISSUE THREAD

**Issue by jlewi** [jlewi/foyle#53]
#51 changed how we setup the loggers in order to write to two different sinks.
As a side effect I think we stopped logging the caller.

**Comment by jlewi** [comment]
Previously we were doing

```
c := zap.NewDevelopmentConfig()
log, err := c.Build()
if err != nil {
    t.Fatalf("Failed to create logger: %v", err)
}
```

**Comment by jlewi** [comment]
It looks like that calls cfg.buildOptions on the writeSyncer
https://github.com/uber-go/zap/blob/3ffa0c00594205c2c623aa9493847217982b558d/config.go#L264

Here we can add the option to AddCaller
https://github.com/uber-go/zap/blob/3ffa0c00594205c2c623aa9493847217982b558d/config.go#L272

**Comment by jlewi** [comment]
This turns out to be a one line fix

```
newLogger = newLogger.WithOptions(zap.AddCaller())
zap.ReplaceGlobals(newLogger)
```

Fix is in #54
